src/modules/translation/languages/en.json

```diff
@@ -1502,4 +1503,4 @@
       "subtitle": "Thank you for revieweing this collection"
     }
   }
-}
+}
```

Comment:
mhmm, is your IDE removing the extra newline in here? I think this change shouldn't be here 😅

Reply:
same for other `.json` files!

Reply:
Thanks for the review! Updated